src/azure-cli/azure/cli/command_modules/cdn/custom/custom_rule_util.py

```diff
@@ -292,7 +292,7 @@ def create_action(action_name, cache_behavior=None, cache_duration=None, header_
         action = {
             "cache_key_query_string": {
                 "parameters": {
-                    "type_name" "DeliveryRuleCacheKeyQueryStringBehaviorActionParameters"
+                    "type_nameDeliveryRuleCacheKeyQueryStringBehaviorActionParameters"
```

Comment:
This original code seems to be wrong.

```suggestion
                    "type_name": "DeliveryRuleCacheKeyQueryStringBehaviorActionParameters"
```

Reply:
Also missing `,` at the end of file.

Previously, `type_nameDeliveryRuleCacheKeyQueryStringBehaviorActionParametersquery_string_behavior` is the first key.